### PR TITLE
fixing issue 13980

### PR DESCRIPTION
**Pull request by sbezverk**
Signed-off-by: Serguei Bezverkhi <sbezverk@cisco.com>

Closes: #13980

### CODE REVIEW

File: tools/packaging/common/istio-iptables.sh [hunk 1]
```diff
@@ -223,9 +223,11 @@ fi
 # need to split them in different arrays one for ipv4 and one for ipv6
 # in order to not to fail
 pl='/*'
+IFS=',' read -ra EXCLUDE <<< "$OUTBOUND_IP_RANGES_EXCLUDE"
+IFS=',' read -ra INCLUDE <<< "$OUTBOUND_IP_RANGES_INCLUDE"
```
Comment (howardjohn) [inline tools/packaging/common/istio-iptables.sh]: I don't really have a clue what this does. not saying it is wrong, I see this in a lot of places so it is probably right, but I can't give a good review.

Maybe we need to port the tests they added in the distroless PR so we can get some confidence on these fixes?

Comment (sbezverk) [reply tools/packaging/common/istio-iptables.sh]: It converts a comma separated string into an array so then each element of this string can be tested individually.

Comment (tiswanso) [reply tools/packaging/common/istio-iptables.sh]: @howardjohn -- could you point at the tests you're referring to?  this pr #12869?

File: tests/e2e/tests/simple/testdata/nginx/nginx.yaml.tmpl [hunk 2]
```diff
@@ -236,7 +236,7 @@ spec:
         traffic.sidecar.istio.io/includeInboundPorts: ""
         # Exclude outbound traffic to kubernetes master from redirection.
         # This is required in order to support single-namespace Istio configurations.
-        traffic.sidecar.istio.io/excludeOutboundIPRanges: "{{.KubeMasterCIDR}}"
+        traffic.sidecar.istio.io/excludeOutboundIPRanges: "{{.KubeMasterCIDR}},1.1.1.1/24,2.2.2.2/16,3.3.3.3/20"
```
Comment (sbezverk) [inline tests/e2e/tests/simple/testdata/nginx/nginx.yaml.tmpl]: Now, e2e-simple test will test parsing a comma separated exclude ranges.

File: tools/packaging/common/istio-iptables.sh (hunk 1 above)
Comment (sbezverk) [reply tools/packaging/common/istio-iptables.sh]: @howardjohn @tiswanso I modified e2e-simple to test parsing of the list of exclude ranges, so this part of code will be tested by a required test.

**Comment by linsun** [comment]
@sbezverk I saw #13980 is already closed, is this PR still needed?

**Comment by sbezverk** [comment]
@linsun Yes, it has highlighted an issue applicable for both ipv4 and ipv6 cases. So we still need to fix it. Please approve.

**Pull request merged**